src/ScottPlot5/ScottPlot5/PlottableFactory.cs

```diff
@@ -24,6 +24,13 @@ public Heatmap Heatmap(double[,] intensities)
         return heatmap;
     }
 
+    public Legend Legend(IList<LegendItem>? legendItems = null)
```

Comment:
Maybe corner should be the first optional argument... Customizing placement when turning the legend on is a pretty common use case.

Reply:
Aye, although I'm not particularly happy with `Position` being a corner enum, I feel like that's going to paint us into a corner.

Perhaps we have an `IPosition` that returns a `Pixel` for a given screen size and legend size, and we only implement it for the four corners for now? That seems like an overkill solution but given that layout code always seems like it should be easier than it is it might be our best bet.

Reply:
Interesting idea!

> That seems like an overkill solution

I do like it, but I'll go ahead and merge this as it is and we can extend this functionality in the future